### PR TITLE
Generate a PDF version of the docsite. Request from a user in Slack.

### PR DESCRIPTION
**Pull request by mikehearn**

### CODE REVIEW

File: docs/make-docsite.sh [hunk 1]
```diff
@@ -11,4 +11,7 @@ else
     source virtualenv/Scripts/activate
 fi
 
+# TODO: The PDF rendering is pretty ugly and can be improved a lot.
```
Comment (tlil) [inline docs/make-docsite.sh]: Yes, let's take this as a future action. The most pressing issue that I can see is overflowing (and consequently cropping of various) code snippets, e.g.:

image

File: docs/source/index.rst [hunk 2]
```diff
@@ -22,8 +22,12 @@ If you have questions or comments, then get in touch on `Slack <https://slack.co
 
 We look forward to seeing what you can do with Corda!
 
+.. note:: You can read this site offline. Either `download the PDF`_ or download the Corda source code, run ``gradle buildDocs`` and you will have
+   a copy of this site in the docs/build/html directory.
```
Comment (tlil) [inline docs/source/index.rst]: Change to ``docs/build/html`` ?

**Comment by mikehearn** [comment]
https://r3-cev.atlassian.net/browse/CORDA-1290 filed

**Comment by tlil** [comment]
Thanks :+1:

**Pull request merged**